app/decorators/placement_decorator.rb

```diff
@@ -10,16 +10,18 @@ def mentor_names
     end
   end
 
-  def subject_names
-    if subjects.any?
-      subjects.pluck(:name).sort.to_sentence
+  def subject_name
+    if additional_subjects.exists? || additional_subjects.present?
```

Comment:
Yes that's right, because we're using the decorator in both the built placement and the saved placement we can get a minor performance gain by using `.exists?` over `.present?` but unfortunately we can't use that if the placement hasn't been saved.

We could just swap it out for `.present?` but it does mean pulling stuff out of the database.